wwwroot/index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="en" class="terria">
 <head>
+    <base href="/">
```

Comment:
Actually now index.html gets updated as part of the `render-datasource-templates` gulp task, should we remove this from source to avoid confusion?